docs/Requirements.md

```diff
@@ -136,4 +223,4 @@ You can install Sequel Ace with:
 3. _Note #1: If you get a "Missing Microsoft Visual C++ 2008 Redistributables" error message while installing OpenSSL, download the ** [Microsoft Visual C++ 2008 Redistributable Package (x64)](http://www.microsoft.com/en-us/download/details.aspx?id=29) ** (1.7MB Installer) and install it. If you need 32bit support, download and install the [ **Microsoft Visual C++ 2008 Redistributable Package (x86)** ](http://www.microsoft.com/en-us/download/details.aspx?id=15336)._
 4. _Note #2: While installing OpenSSL, choose **The OpenSSL binaries (/bin) directory** (NOT "The Windows system directory") when given the choice on where to copy the OpenSSL DLLs. These DLLs will need to be located easily for Core Installation._
 
-***
+5. If you don't get the MySQL 5.7 library, you can download it from the following [link](https://cdn.discordapp.com/attachments/265262398942740481/640628930117894154/lib_mysql_5_7_28_win.zip)
```

Comment:
I dont like this.

because the .lib must have the EXACT version that your MySQL Server has or it will not work. 
For example if you have 5.7.1 and the DLL is 5.7.2 it won't work, (Dummy versions ofc)

So attaching this is bad when the user can just go into their MySQL file and get it.

Reply:
Info on how you get the file is at https://github.com/azerothcore/wiki/blob/master/docs/Installation.md

Reply:
Ok, now I eliminate it then.

Reply:
Thanks good job!